<launch>

  <!-- start the node "move_camera" which provides the services "start_move_camera" and "stop_move_camera" -->
  <node name="move_camera_c" pkg="motion_control" type="move_camera_c" respawn="true" output="screen">
  </node>

  <!-- start the node "pcl_merger" which provides the services "start_pcl_merge" and "stop_pcl_merge" -->
  <node name="pcl_merger_b" pkg="object_localizer" type="pcl_merger_b" respawn="true" output="screen">
  </node>

</launch>
